src/Microsoft.Azure.WebJobs.Host/Loggers/FunctionInstanceLogger.cs

```diff
@@ -47,7 +48,12 @@ public Task LogFunctionCompletedAsync(FunctionCompletedMessage message, Cancella
         private void Log(LogLevel level, FunctionDescriptor descriptor, Guid functionId, string message, Exception exception = null)
         {
             ILogger logger = _loggerFactory?.CreateLogger(LogCategories.CreateFunctionCategory(descriptor.LogName));
-            logger?.Log(level, 0, message, exception, (s, e) => s);
+            var logData = new Dictionary<string, object>
+            {
+                ["Summary"] = message,
+                [ScopeKeys.FunctionInvocationId] = functionId.ToString()
```

Comment:
Recommend either creating a new LogPropertyKeys.cs class peer to LogCategories/ScopeKeys, or adding new constants for log properties (distinct from ScopeKeys) to the existing LogConstants class. @brettsam might also know whether there isn't already a set of constants for these defined somewhere

Reply:
Will add a constant for Summary.  Earlier logstate was just a string message, now including functionInviocationId in the log state so that systemlogger in the functions runtime can access it.
@brettsam - Let me know if there is a better way to do this.